[2023-10-09T21:27:35.908+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: youtube_data_pipeline.scrape_data manual__2023-10-09T21:13:14.360072+00:00 [queued]>
[2023-10-09T21:27:35.931+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: youtube_data_pipeline.scrape_data manual__2023-10-09T21:13:14.360072+00:00 [queued]>
[2023-10-09T21:27:35.934+0000] {taskinstance.py:1361} INFO - Starting attempt 2 of 2
[2023-10-09T21:27:35.965+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): scrape_data> on 2023-10-09 21:13:14.360072+00:00
[2023-10-09T21:27:35.980+0000] {standard_task_runner.py:57} INFO - Started process 1101 to run task
[2023-10-09T21:27:35.991+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'youtube_data_pipeline', 'scrape_data', 'manual__2023-10-09T21:13:14.360072+00:00', '--job-id', '18', '--raw', '--subdir', 'DAGS_FOLDER/.ipynb_checkpoints/youtube-checkpoint.py', '--cfg-path', '/tmp/tmp93y4xty5']
[2023-10-09T21:27:36.001+0000] {standard_task_runner.py:85} INFO - Job 18: Subtask scrape_data
[2023-10-09T21:27:36.111+0000] {task_command.py:415} INFO - Running <TaskInstance: youtube_data_pipeline.scrape_data manual__2023-10-09T21:13:14.360072+00:00 [running]> on host 4e554196f202
[2023-10-09T21:27:36.306+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='MSc DS-7029-20275335' AIRFLOW_CTX_DAG_ID='youtube_data_pipeline' AIRFLOW_CTX_TASK_ID='scrape_data' AIRFLOW_CTX_EXECUTION_DATE='2023-10-09T21:13:14.360072+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-10-09T21:13:14.360072+00:00'
[2023-10-09T21:27:42.910+0000] {http.py:140} WARNING - Encountered 403 Forbidden with reason "commentsDisabled"
[2023-10-09T21:27:42.916+0000] {taskinstance.py:1943} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/.ipynb_checkpoints/youtube-checkpoint.py", line 145, in scrape_data
    video_comment_response = youtube.commentThreads().list(
  File "/home/airflow/.local/lib/python3.8/site-packages/googleapiclient/_helpers.py", line 130, in positional_wrapper
    return wrapped(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/googleapiclient/http.py", line 938, in execute
    raise HttpError(resp, content, uri=self.uri)
googleapiclient.errors.HttpError: <HttpError 403 when requesting https://youtube.googleapis.com/youtube/v3/commentThreads?part=snippet&videoId=QWUDHqYKaG8&textFormat=plainText&key=AIzaSyD-uO1hyk2x-gzoWrdByiwFqaCZuS1wwF8&alt=json returned "The video identified by the <code><a href="/youtube/v3/docs/commentThreads/list#videoId">videoId</a></code> parameter has disabled comments.". Details: "[{'message': 'The video identified by the <code><a href="/youtube/v3/docs/commentThreads/list#videoId">videoId</a></code> parameter has disabled comments.', 'domain': 'youtube.commentThread', 'reason': 'commentsDisabled', 'location': 'videoId', 'locationType': 'parameter'}]">
[2023-10-09T21:27:42.988+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=youtube_data_pipeline, task_id=scrape_data, execution_date=20231009T211314, start_date=20231009T212735, end_date=20231009T212742
[2023-10-09T21:27:43.065+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 18 for task scrape_data (<HttpError 403 when requesting https://youtube.googleapis.com/youtube/v3/commentThreads?part=snippet&videoId=QWUDHqYKaG8&textFormat=plainText&key=AIzaSyD-uO1hyk2x-gzoWrdByiwFqaCZuS1wwF8&alt=json returned "The video identified by the <code><a href="/youtube/v3/docs/commentThreads/list#videoId">videoId</a></code> parameter has disabled comments.". Details: "[{'message': 'The video identified by the <code><a href="/youtube/v3/docs/commentThreads/list#videoId">videoId</a></code> parameter has disabled comments.', 'domain': 'youtube.commentThread', 'reason': 'commentsDisabled', 'location': 'videoId', 'locationType': 'parameter'}]">; 1101)
[2023-10-09T21:27:43.146+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-10-09T21:27:43.203+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
